sycl/source/detail/scheduler/graph_builder.cpp

```diff
@@ -1028,18 +1028,18 @@ void Scheduler::GraphBuilder::cleanupCommandsForRecord(
     AllocaCmd->MUsers.clear();
   }
 
-  // Linked alloca's share dependencies. Unchain from deps linked alloca's.
-  // Any cmd of the alloca - linked_alloca may be used later on.
+  // Make sure the Linked Allocas are marked visited by the previous walk.
+  // Remove all dependencies AllocaCommands has on other commands.
```

Comment:
This line should be something like "Remove allocation commands from the users of their dependencies"

Reply:
Fixed.